project/Li-Pro/README.rst

```diff
@@ -0,0 +1,77 @@
+========================
+Columnar Structure Array
+========================
+
+--------
+Abstract
+--------
+
+An **array of structure (AoS)** is not cache-friendly nor SIMD friendly. One of its replacement is **structure of array (SoA)**.
+However, AoS is so straightforward for programmers to code in, and easy to maintain. It is also fundamental for OOP.
+
+This project designs a structure helper for programmers to use SoA, but to code as in AoS.
```

Comment:
There are a couple of attempts to use SoA data structure with API similar to AoS and the work will be larger than the scope of 8 weeks.  It will be more complex when you factor in the use of an interpreter.  I will suggest to exclude the SoA/AoS conversion from the project.